lang/en.lyaml

```diff
@@ -1129,6 +1129,7 @@ en:
         buildSucceededAutomaticallyDeploying: "Build #{{ buildId }} succeeded. {{#bold}}Automatically deploying{{/bold}} to {{ accountIdentifier }}\n"
         cleanedUpTempFile: "Cleaned up temporary file {{ path }}"
         viewDeploys: "View all deploys for this project in HubSpot"
+        unableToFindAutodeployStatus: "Unable to find the auto deploy for build #{{ buildId }}. This deploy may have been skipped. {{ viewDeploysLink }}."
```

Comment:
Can we remove the entry for this under `pollFetchProject`?

Reply:
Yep, for some reason I thought it was being used elsewhere. https://github.com/HubSpot/hubspot-cli/pull/1355/commits/6f19dcb27688017e1d6bd466bc7370cd5be6881b